packages/node_modules/cerebral/package.json

```diff
@@ -19,7 +19,12 @@
   "homepage": "http://cerebral.github.io/cerebral-website",
   "dependencies": {
     "eventemitter3": "^2.0.2",
-    "function-tree": "next"
+    "function-tree": "next",
+    "@cerebral/react": "next",
+    "@cerebral/inferno": "next",
+    "@cerebral/preact": "next",
+    "@cerebral/vue": "next",
+    "@cerebral/angularjs": "next"
```

Comment:
Should not be dependencies? The views are optional... might not even add a view :-)

Reply:
I thing we need these imports so we can reexport them while the old imports are deprecated.
They will be removed when deprecation will be removed.